templates/atlus_fbn.bt

```diff
@@ -0,0 +1,143 @@
+typedef ubyte u8;
+typedef short s16;
+typedef ushort u16;
+typedef uint u32;
+typedef float f32;
+
+typedef enum<u16>
+{
+	NPCSpawnPos = 4,// Not sure but this is what I could get from my research
+	NPC = 5,
+} MajorType;
+
+typedef struct
+{
+	u8 Unknown0;
+	u8 Unknown1;
+	u8 Unknown2;
+	u8 Unknown3;
+	MajorType Major <name = "Major">;
+	s16 Minor <name = "Minor">;
+	u8 Res_ID <name = "Res ID">;
+	u8 Unknown;
+	u16 Field000;
+	float CollidRadius <name = "Collision Radius">;
+	float Field002;
+	float Field003;
+	float Field004;
+	float Field005;
+	float Field006 ;
+	float Field007;
+	float Field008;
+	float Field009;
+	float Field010;
+	float Field011 ;
+	float Field012;
+	float Field013;
+	float Field014;
+	float Field015;
+	float Field016 ;
+	float Field017;
+	float YPos <name = "Y Pos">;
+	float Field019;
+	float XPos <name = "X Pos">;
+	float Field021;
+	u32 ShadowType <name = "Shadow Type">;
+	float ShadowRadius <name = "Shadow Radius">;
+	u8 Unknown4;
+	u8 Unknown5;
+	u8 Unknown6;
+	u8 Unknown7;
+	u8 Unknown8;
+	u8 Unknown9;
+	u8 Unknown10;
+	u8 Unknown11;
+	if (Type1EntrySize==320)
```

Comment:
This variable doesn't exist in this scope, you should pass it to the struct constructor if you wish to use it